app/components/TextEditor/TextEditor.tsx

```diff
@@ -18,8 +21,16 @@ const TextEditor = (): React.ReactElement => {
 
   const saveNote = () => {
     const content = editor?.getJSON();
+    if (!content) return;
+
+    const note = {
+      id: crypto.randomUUID(),
+      content,
+      dateAdded: new Date(),
+      dateUpdated: new Date(),
```

Comment:
Same note as above - I have a feeling that creating the `id` and date data on the frontend is an anti-pattern of some type. 

Plus if we're not using the class to do this work, why even have them?